libbeat/publisher/pipeline/client.go

```diff
@@ -203,12 +203,10 @@ func (c *client) onPublished() {
 }
 
 func (c *client) onFilteredOut(e beat.Event) {
-	c.logger.Debugf("Pipeline client receives callback 'onFilteredOut' for event: %+v", e)
```

Comment:
I'm not sure I agree with this change. I remember this log line helped in a few SDHs. Is there a way to log it once? The infinite loop you mentioned in the description – is it expected behaviour? Because this change only removes observability of that loop not the loop itself, right?

Reply:
Not that I'm aware of. Not as callbacks called, we do have some metrics for events dropped, I'm not sure about filtered out.

Reply:
Okay, at least we have it from dropped. LGTM.